dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/action/ImportContentletsAction.java

```diff
@@ -308,21 +322,25 @@ else if(eCode != null)
 	 */
 	private void detectEncodeType(final HttpSession session, final File file) throws IOException {
 
-		String encodeType=null;
+		String encodeType   = null;
 
 		if (null != file && file.exists()) {
             byte[] buf = new byte[4096];
-            FileInputStream fis = new FileInputStream(file);
-            UniversalDetector detector = new UniversalDetector(null);
-            int nread;
-            while ((nread = fis.read(buf)) > 0 && !detector.isDone()) {
-                detector.handleData(buf, 0, nread);
-            }
-            detector.dataEnd();
-            encodeType = detector.getDetectedCharset();
-            session.setAttribute(ENCODE_TYPE, encodeType);
-            detector.reset();
-            fis.close();
+            try (FileInputStream fis = new FileInputStream(file)){
+
+
+				UniversalDetector detector = new UniversalDetector(null);
+				int nread;
+				while ((nread = fis.read(buf)) > 0 && !detector.isDone()) {
+					detector.handleData(buf, 0, nread);
+				}
+				detector.dataEnd();
+				encodeType = detector.getDetectedCharset();
+				session.setAttribute(ENCODE_TYPE, encodeType);
+				detector.reset();
+			}catch (IOException e){
+				throw e;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add logic to this catch clause or eliminate it and rethrow the exception automatically. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2737)